core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceBindingBuilder.java

```diff
@@ -81,11 +81,21 @@ public AnnotatedServiceBindingBuilder pathPrefix(String pathPrefix) {
      * Adds the given {@link ExceptionHandlerFunction} to this {@link AnnotatedServiceBindingBuilder}.
      */
     public AnnotatedServiceBindingBuilder exceptionHandler(ExceptionHandlerFunction exceptionHandlerFunction) {
-        requireNonNull(exceptionHandlerFunction, "exceptionHandler");
+        requireNonNull(exceptionHandlerFunction, "exceptionHandlerFunctions");
         exceptionHandlerFunctionBuilder.add(exceptionHandlerFunction);
         return this;
     }
```

Comment:
- How about adding `exceptionHandlers(ExceptionHandlerFunction...)` and deprecating this method?